db/migrate/20160825190205_devise_create_users.rb

```diff
@@ -0,0 +1,41 @@
+class DeviseCreateUsers < ActiveRecord::Migration[5.0]
+  def change
+    create_table :users do |t|
+      ## Database authenticatable
+      t.string :email,              null: false, default: ''
+      t.string :encrypted_password, null: false, default: ''
```

Comment:
Does that database store an encrypted version of the password instead of a password hash?  Or is this just poor variable naming / devise auto-generated?